server/protocols/webadmin/webadmin-data/src/main/java/org/apache/james/webadmin/routes/MappingRoutes.java

```diff
@@ -77,15 +107,13 @@ public void define(Service service) {
         }
     }
 
-    private List<MappingValue> toMappingValues(Map.Entry<MappingSource, Mappings> entry) {
+    private List<MappingValue> toMappingValues(MultiMap.Entry<MappingSource, Mappings> entry) {
```

Comment:
That is not what I meant. I meant use a Guavate.toXXMultimap(...) line 68

Reply:
Forget about this, I now understand why you did this (the stream elements already are mapping collection)